RNGestureHandlerModule.web.js

```diff
@@ -0,0 +1,698 @@
+import Hammer from 'hammerjs';
+import { findNodeHandle } from 'react-native';
+import State from './State';
+
+const MULTI_FINGER_PAN_MAX_PINCH_THRESHOLD = 0.1;
+const MULTI_FINGER_PAN_MAX_ROTATION_THRESHOLD = 7;
+const DEG_RAD = Math.PI / 180;
+
+// Used for sending data to a callback or AnimatedEvent
+function invokeNullableMethod(name, method, event) {
```

Comment:
This function is a little long-winded to handle Reanimated. It would be nice if Reanimated sent back a function and did it's own value merging internally like RN.Animated.